File: app/src/main/java/net/squanchy/service/firebase/model/FirebaseEvent.java
package net.squanchy.service.firebase.model;

import android.support.annotation.Nullable;

import java.util.List;

public class FirebaseEvent {

    public String id;

    public String name;

    public String type;

    public String short_description;

    @Nullable
    public String description;

    public Long start_time;

    public Long end_time;

    public String day_id;

    @Nullable
    public String track_id;

    @Nullable
    public String experience_level;

    @Nullable
    public String place_id;

    @Nullable
    public List<String> speaker_ids;

    @Nullable
    public FirebaseLinks links;
}

File: app/src/main/java/net/squanchy/service/firebase/model/FirebaseEvent.kt
package net.squanchy.service.firebase.model

data class FirebaseEvent(
    var id: String? = null,
    var name: String? = null,
    var type: String? = null,
    var start_time: Long? = null,
    var end_time: Long? = null,
    var day_id: String? = null,
    var description: String? = null,
    var track_id: String? = null,
    var experience_level: String? = null,
    var place_id: String? = null,
    var speaker_ids: List<String>? = null,
    var links: FirebaseLinks? = null
)
